pyattest/verifiers/google_play_integrity_attestation.py

```diff
@@ -0,0 +1,57 @@
+import base64
+import json
+from typing import Optional
+
+from jose import jwe, jws
+
+from pyattest.exceptions import PyAttestException, InvalidNonceException, \
+    InvalidAppIdException, InvalidAppIntegrity, \
+    InvalidDeviceIntegrity
+from pyattest.verifiers.attestation import AttestationVerifier
+
+
+class GooglePlayIntegrityAttestationVerifier(AttestationVerifier):
+    def verify(self):
+        """
+        Verify the given attestation based on the Google documentation. The attestation is a nested JWS in JWE object.
+
+        Raises a PyAttestException as soon as one of the verification steps fails.
+
+        See also:
+        Google Documentation https://developer.android.com/google/play/integrity/verdict#decrypt-verify
+        """
+        payload = self.unpack(self.attestation.raw)
+        self.check_request_details(payload.get('requestDetails'))
+        if self.attestation.config.production:
+            self.check_app_integrity(payload.get('appIntegrity'))
+            self.check_device_integrity(payload.get('deviceIntegrity'))
+
+    def unpack(self, jwt_object: str) -> dict:
```

Comment:
According to https://developer.android.com/google/play/integrity/verdict#decrypt-verify-google-servers it is recommended to decrypt and verify the jwt via Google's service. Is there a reason why we're not doing that?